amazon-s3/5.7/modules/ROOT/pages/index.adoc

```diff
@@ -10,35 +10,64 @@ Instant access to the Amazon S3 API enables seamless integrations between Amazon
 AWS SDK for Java provides a Java API for AWS infrastructure services.
 The Amazon S3 connector is built using the SDK for Java.
 
-Release Notes: xref:release-notes::connector/amazon-s3-connector-release-notes-mule-4.adoc[Amazon S3 Connector Release Notes] +
-Exchange: https://anypoint.mulesoft.com/exchange/com.mulesoft.connectors/mule-amazon-s3-connector[Amazon S3 connector in Anypoint Exchange]
+For compatibility information and fixed issues, see the xref:release-notes::connector/amazon-s3-connector-release-notes-mule-4.adoc[Amazon S3 Connector Release Notes].
+
+For a major or minor release, see xref:amazon-s3-connector-upgrade-migrate.adoc[Upgrade and Migrate].
 
 == About Connectors
 
 Anypoint connectors are Mule runtime engine extensions that enable you to connect to APIs and resources on external systems, such as Salesforce, Amazon S3, ServiceNow, and Twitter.
 
 == Prerequisites
 
-Before creating an app, you must have access to the Amazon S3 target resource, Amazon Web Services, and Anypoint Platform. You must also understand how to create a Mule app using Design Center or Anypoint Studio, and have AWS Identity and Access Management (IAM) credentials.
+To use this connector, you must be familiar with:
+
+* The Amazon S3 API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access the Amazon S3 target resource
+* Anypoint Platform
+* Anypoint Studio version 7.x or later
+* Amazon Web Services
+* AWS Identity and Access Management (IAM) credentials
 
 [NOTE]
 For the Amazon S3 operations to work, you need to enable or update the subset of the overall list of actions in the Amazon S3 bucket to specify that the AWS account has access these actions.
```

Comment:
The "For" looks indented.

Reply:
Weird... it appears fine in the local build... but good eye.